### PR TITLE
Changes to "Distribution Status" switch [PD-3430]

### PR DESCRIPTION
**Pull request by tdphillips**
## What's this PR do?
- Hides switch if the user is viewing a single organization
- Adds a confirm when disabling the switch
## Screenshots (if appropriate)
2017-07-07-145104_819x432_scrot
## How should this be manually tested?
Select an organization. In any of that organization's subscreens, the switch should not be present.
Head back to your main TJD screen. Switch off distribution. You should see the pictured dialog.
## Any background context you want to provide?
The text is not my own. If anyone has suggestions, I'm all ears.

### CODE REVIEW

**Comment by aljordan82** [comment]
The code looks fine, however, I couldn't test it locally.

File: gulp/src/cbo/components/commonCbo/Sidebar.js [hunk 1]
```diff
@@ -84,16 +96,20 @@ class Sidebar extends Component {
...
         <a className="btn leave-feedback" href="http://www.directemployers.org/beta-feedback" target="_blank">Leave Us Feedback</a>
-          <h2 className="top">Organization Status</h2>
-          <div id="toggle_container">
-            <p className="distribution-status-title">Distribution Status</p>
-            <div className="toggle-inner-container">
-              <span className="toggle-main-container">
-                <input type="checkbox" className="toggle-input" id="toggle_input" name="check" checked={checked} onChange={(e) => this.updateCboStatus(e)} />
-                <label data-on="ON" data-off="OFF" htmlFor="toggle_input" className="toggle-label"></label>
-              </span>
+        {viewTJD ? '' :
```
Comment (koonsies) [inline gulp/src/cbo/components/commonCbo/Sidebar.js]: It looks like we want to restrict this control to display only on the main page. That is similar to the check on line 70 (82) .. perhaps we could use the same logic in both places?

Comment (tdphillips) [inline gulp/src/cbo/components/commonCbo/Sidebar.js]: Thanks. The way I did it was leaving the switch on the "new organization" form, which probably wasn't ideal.

**Comment by tdphillips** [comment]
The page no longer reloads if you click cancel or otherwise dismiss the modal.

File: gulp/src/cbo/components/commonCbo/Sidebar.js [hunk 2]
```diff
@@ -26,10 +27,18 @@ class Sidebar extends Component {
...
   }
-  updateCboStatus(e) {
+  async updateCboStatus(e) {
+    e.preventDefault();
     const {dispatch} = this.props;
     const checked = e.target.checked;
+    const message = 'Turning off distribution means your jobs will no longer be delivered to the listed organizations. Are you sure that is the action you want to take?';
+    if (!checked) {
+      if (!(await runConfirmInPlace(dispatch, message))) {
+        return false;
```
Comment (aljordan82) [inline gulp/src/cbo/components/commonCbo/Sidebar.js]: just `return;` is better

Comment (tdphillips) [inline gulp/src/cbo/components/commonCbo/Sidebar.js]: Why?

Comment (aljordan82) [inline gulp/src/cbo/components/commonCbo/Sidebar.js]: Nothing uses the return values, so it is better to keep the return value consistent

Comment (tdphillips) [inline gulp/src/cbo/components/commonCbo/Sidebar.js]: What about the `return true` a few lines later? That seems inconsistent with all other return values.

**Pull request merged**